presto-main/src/main/java/com/facebook/presto/sql/planner/SplitSourceFactory.java

```diff
@@ -135,6 +137,17 @@ private Visitor(Session session, StageExecutionDescriptor stageExecutionDescript
             return ImmutableMap.of(node.getId(), splitSource);
         }
 
+        private SplitSchedulingStrategy getSplitSchedulingStrategy(StageExecutionDescriptor stageExecutionDescriptor, PlanNodeId scanNodeId)
```

Comment:
static?

Reply:
This method is inside a non-static inner class :(